packages/jmix-front-generator/src/generators/react-typescript/app/template/src/i18n/i18nInit.ts.ejs

```diff
@@ -0,0 +1,55 @@
+<% if (isLocaleUsed('en')) { -%>
+  import en from './en.json';
+  import enUs from 'antd/es/locale/en_US';
+<% } -%>
+<% if (isLocaleUsed('fr')) { -%>
+  import fr from './fr.json';
+  import frFr from 'antd/es/locale/fr_FR';
+<% } -%>
+<% if (isLocaleUsed('ru')) { -%>
+  import ru from './ru.json';
+  import ruRu from 'antd/es/locale/ru_RU';
+<% } -%>
+import { localesStore } from "@haulmont/jmix-react-web";
+import { antdLocalesStore } from "@haulmont/jmix-react-antd"
+
+export function initializeLocales() {
+  <% if (isLocaleUsed('en')) { -%>
+    localesStore.addLocale({
+      locale: 'en',
+      caption: "English",
+      messages: en
+    });
+
+    antdLocalesStore.addAntdLocale({
+      localeName: 'en',
+      antdLocale: enUs
+    })
+  <% } -%>
+
+  <% if (isLocaleUsed('fr')) { -%>
+    localesStore.addLocale({
+      locale: 'fr',
+      caption: "Français",
+      messages: fr
+    });
+
+    antdLocalesStore.addAntdLocale({
+      localeName: 'fr',
+      antdLocale: frFr
+    })
+  <% } -%>
+  <% if (isLocaleUsed('ru')) { -%>
+    localesStore.addLocale({
+      locale: 'ru',
+      caption: "Русский",
+      messages: ru
+    });
+
+    antdLocalesStore.addAntdLocale({
+      localeName: 'ru',
+      antdLocale: ruRu
+    })
+
+  <% } -%>
```

Comment:
Just noticed that `cn-zh` locale is missing.

Also, let's imagine we want to add a new locale out-of-the-box. We will add a new `.json` file and add an entry to `SUPPORTED_CLIENT_LOCALES`. Ideally that should be it. How can we avoid having to put any new locale manually into this file?